besu/src/main/java/org/hyperledger/besu/cli/options/stable/P2PTLSConfigOptions.java

```diff
@@ -103,6 +104,13 @@ public class P2PTLSConfigOptions {
           "Whether to send a SNI header in the TLS ClientHello message (default: ${DEFAULT-VALUE})")
   private final Boolean p2pTlsClientHelloSniHeaderEnabled = false;
 
+  @Option(
```

Comment:
This is in the wrong place, this option doesn't have anything to do with TLS. Better place for this would be `SynchronizerOptions`

Reply:
Sure, I'll move it over